frontend/text-editor/src/editor/content/dom/TextNode.js

```
@@ -60,5 +60,5 @@ export function getClosestTextNode(node) {
   if (isInline(node)) return node.firstChild;
   if (isParagraph(node)) return node.firstChild.firstChild;
   if (isRoot(node)) return node.firstChild.firstChild.firstChild;
-  throw new Error("Cannot find a text node");
+  return null;
```

This should `throw` to prevent silent errors.